lib/commands/clear.js

```diff
@@ -15,16 +15,15 @@ module.exports = async function clear(context, selector) {
     throw errorWithUsefulStack;
   }
 
-  await context.page.evaluate(selectorEl => {
-    const el = document.querySelector(selectorEl);
+  await context.page.evaluate(el => {
     el.focus();
     if (el.setSelectionRange) {
       const elType = el.getAttribute('type');
       el.setAttribute('type', 'text'); // setSelectionRange is not available for input[type="email"] and input[type="number"]
       el.setSelectionRange(-1, -1);
       el.setAttribute('type', elType);
     }
-  }, selector);
+  }, elem);
```

Comment:
Elements found by `page.$` method can be passed to page.evaluate callback.

> ElementHandle instances can be used as arguments in page.$eval() and page.evaluate() methods.

https://pptr.dev/#?product=Puppeteer&version=v5.5.0&show=api-class-elementhandle